test-suite/output/PrintKeywords.out

```diff
@@ -0,0 +1,95 @@
+!
+#
+#[
+%
+&
```

Comment:
If, say, `#[` is a keyword, what limitations apply (since it's obviously not an identifier)?  Are you able to use that in a notation definition?  The doc should state the limitation.  Also the doc should probably refer to "keywords and other tokens", the terminology used in `basic.rst`.

Reply:
I think the link addresses different points, namely that Coq uses the longest match and won't break up identifiers.  How about adding a statement (if true) that a) tokens can't be part of the syntax defined by a notation nor b) can you define new notation tokens that have a token as a prefix, e.g. `#[!` won't work because it begins with the token `#[`.

Reply:
The other doc changes look good.

Reply:
neither of those is true
The link addresses that if you write `#[` it will never be parsed as `#` followe by `[` because `#[` is registered as a token.

Reply:
Ah, I forgot about the quoting with apostrophes.  You would need something like `Notation "A '#[!' B" := (iff A B) (at level 0).`  That would be a good thing for us to mention in the `Notation` documentation.

Reply:
We should also mention that `Notation` can create new keywords, e.g.

`Notation "A 'IFF' B" := (iff A B) (at level 0).`